libsoftwarecontainer/src/gateway/dbus/dbusgatewayinstance.cpp

```diff
@@ -123,20 +134,26 @@ bool DBusGatewayInstance::activateGateway()
     return testDBusConnection(config);
 }
 
-bool DBusGatewayInstance::testDBusConnection(const std::string &config)
+bool DBusGatewayInstance::testDBusConnection(const std::string &cfg)
```

Comment:
I saw the usage when I wrote the comment and was torn between just deleting the comment and keeping it because I understand your reasoning ;). 

Perhaps configuration could work? or stringConfig?